core/modules/user/user.module

```diff
@@ -453,6 +453,23 @@ function user_password($length = 10) {
   return $pass;
 }
 
+/**
+ * Checks if a user role has a permission.
+ *
+ * @param string $role_name
+ *   The role machine name.
+ * @param string $permission_name
+ *   The permission to check.
+ *
+ * @return boolean
+ *   TRUE if the role has the permission, FALSE if it does not have the
+ *   permission.
+ */
+function user_role_has_permission(string $role_name, string $permission_name) {
+  $role = user_role_load($role_name);
+  return in_array($permission, $role->permissions);
```

Comment:
Your parameter is called `$permission_name`, yet here you are using an undefined variable called `$permission`.

Reply:
Thanks @argiepiano - done